specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-05-01-preview/containerregistry_scopemap.json

```diff
@@ -0,0 +1,916 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "TokenList": {
+            "$ref": "./examples/TokenList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "GenerateCredentialsParameters": {
+      "description": "The parameters used to generate credentials for a specified token or user of a container registry.",
+      "type": "object",
+      "properties": {
+        "tokenId": {
+          "description": "The resource ID of the token for which credentials have to be generated.",
+          "type": "string"
+        },
+        "expiry": {
+          "format": "date-time",
+          "description": "The expiry date of the generated credentials after which the credentials become invalid.",
+          "default": "9999-12-31T15:59:59.9999999-08:00",
+          "type": "string"
+        },
+        "name": {
+          "description": "Specifies name of the password which should be regenerated if any -- password or password1.",
+          "enum": [
+            "password",
+            "password2"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PasswordName",
+            "modelAsString": false
+          }
+        },
+        "publicCertificateAsPEM": {
+          "description": "Base 64 encoded string of the public certificate in PEM format that will be used for authenticating the token.",
+          "type": "string"
+        }
+      }
+    },
+    "GenerateCredentialsResult": {
+      "description": "The response from the GenerateCredentials operation.",
+      "type": "object",
+      "properties": {
+        "username": {
+          "description": "The username for a container registry.",
+          "type": "string"
+        },
+        "passwords": {
+          "description": "The list of passwords for a container registry.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TokenPassword"
+          }
+        },
+        "certificate": {
+          "$ref": "#/definitions/TokenCertificate",
+          "description": "The public certificate that will be used for authenticating the token of a container registry."
+        }
+      }
+    },
+    "TokenPassword": {
+      "description": "The password that will be used for authenticating the token of a container registry.",
+      "type": "object",
+      "properties": {
+        "expiry": {
+          "format": "date-time",
+          "description": "The expiry datetime of the password.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The password name \"password\" or \"password1\"",
+          "enum": [
+            "password",
+            "password2"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PasswordName",
+            "modelAsString": false
```

Comment:
See prior comment on modelAsString: true

Reply:
Fixed it